src/main/rx/fport.c

```diff
@@ -335,6 +337,8 @@ static bool fportProcessFrame(const rxRuntimeConfig_t *rxRuntimeConfig)
     UNUSED(rxRuntimeConfig);
 
 #if defined(USE_TELEMETRY_SMARTPORT)
+    static uint8_t consecutiveSensorCount = 0;
```

Comment:
This static do not need initialization, just to avoid static initialization overhead